stacks/api/bucket/BUILD.bazel

```diff
@@ -11,14 +11,15 @@ ts_library(
         "bucket.module.ts",
         "bucket.schema.resolver.ts",
         "bucket.schema.types.ts",
-        "bucket.service.ts",
-        "bucket.ts",
         "bucket-data.controller.ts",
         "bucket-data.service.ts",
         "cache.ts",
         "index.ts",
         "scheduler.ts",
-    ],
+    ] + glob(
+        ["hook/*.ts"],
+        exclude = ["hook/*.spec.ts"],
+    ),
```

Comment:
Since we have the services separated from the main bucket module, we can make the hook module a separate bazel target and depend on it.

This way we will enforce the separation and more incrementality.